cvmfs/cvmfs_config

```diff
@@ -1144,7 +1155,7 @@ cvmfs_killall() {
   echo "OK"
 
   echo -n "Terminating cvmfs2 processes... "
-  killall -q -KILL ${INSTALL_BASE}/bin/cvmfs2
```

Comment:
@jblomer this is a merge conflict. Is this solution compatible with the one submitted in https://github.com/cvmfs/cvmfs/pull/1438?